tutorials/turtlebot4_navigator.md

```diff
@@ -1300,4 +1300,317 @@ else:
 </figure>
 
 {% endtab %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## LLM Integration
+
+This example demonstrates an integration between the OpenAI [Chat Completions API](https://platform.openai.com/docs/guides/gpt/chat-completions-api) and the Turtlebot 4, substantially inspired by the ['Code as Policies: Language Model Programs for Embodied Control' (Liang et al.)](https://arxiv.org/abs/2209.07753) research.
+
+It demonstrates how an LLM can be 'taught' how to use the Turtlebot 4 Navigation API with only a few examples, and also shows how an LLM can easily bridge natural language commands to API calls without an intermediate parsing step.
+
+Before using this as the basis for a product, please consult the substantial list of caveats found in the original paper! :smile:
+
+### Installation
+1. Create an [OpenAI account](https://platform.openai.com/signup?launch) and [API key](https://platform.openai.com/account/api-keys). Be sure to copy your API key to a safe place, it cannot be retrieved after creation!
+2. Verify that your OpenAI account has [some credit](https://platform.openai.com/account/usage). A few cents are all that's required for initial testing
+3. Install the OpenAI Python library via `pip install openai`
+4. If you do not already have a workspace, open a terminal and create one `mkdir ~/turtlebot4_ws/src -p`
+5. Download or clone [the code](https://github.com/turtlebot/turtlebot4_tutorials) into your workspace.
+6. Run colcon build in your workspace `cd ~/turtlebot4_ws/ && colcon build`
+
+### Running
+
+To run this example, first start the Gazebo simulation:
+
+```bash
+ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py nav2:=true slam:=false localization:=true rviz:=true
```

Comment:
This line is only for humble, can we add tabbing to indicate the correct command for galactic as well? (like other tutorials on this page)

Additionally, the humble sim launches with the warehouse world not the depot world so we also need to pass the world and map parameters if we want it to run in depot (so that the destinations make sense)

Reply:
What would the full command for the humble world be? I'll tab it properly